example/generator/GeneratorMetadataProvider.js

```diff
@@ -28,6 +28,15 @@ define([
                         domain: 2
                     }
                 },
+                {
+                    key: "local",
+                    name: "Time",
+                    format: "local-format",
```

Comment:
Changed to "local-format" as "local" wasn't formatting.

Reply:
Oh, yeah, I made a comment about this and then deleted it because I realized that the local formatter is only available when the local time system is installed, which it's not by default 😄 So we'll have to stick with `utc` for now.

Reply:
Which, actually, makes me think we just shouldn't have `local` values at all, because they will be shown in the UI. The best thing to do here might be to set it to use `local-format`, but comment out the metadata for `local`. Make a note about needing to enable the local time system to make use of it.

It's the one time I'll allow commented code to be checked in 😄